uber/templates/static_views/prereg_not_yet_open.html

```diff
@@ -13,7 +13,7 @@ <h1 style='color:red'>{{ c.EVENT_NAME_AND_YEAR }} pre-registration is not yet op
   {%- if c.DEALER_REG_START and c.DEALER_REG_PUBLIC and c.BEFORE_DEALER_REG_SHUTDOWN -%}
   <p>
     <strong>Dealers:</strong>
-    {% if c.BEFORE_DEALER_REG_PUBLIC %}
+    {% if c.BEFORE_DEALER_REG_START %}
```

Comment:
Option 1. will work as long as dealer_reg_public is not set to an empty value. Option 2. is not a good idea, because we now have two variables controlling the same thing.